src/ScrollList/index.tsx

```diff
@@ -35,15 +35,22 @@ export const CustomScrollContainer = React.forwardRef<HTMLDivElement, IFixedSize
 interface IFixedSizeListProps extends Omit<ReactWindow.FixedSizeListProps, 'height' | 'width'> {
   className?: string;
   maxRows?: number;
+  scrollToItem?: number;
 }
 
 const FixedSizeList: React.FunctionComponent<IFixedSizeListProps> = React.forwardRef<
   ReactWindow.FixedSizeList,
   IFixedSizeListProps
 >(function FixedSizeList(props, ref) {
-  const { className, children, itemSize, itemCount, maxRows, style, ...rest } = props;
+  const { className, children, itemSize, itemCount, maxRows, style, scrollToItem, ...rest } = props;
   const listHeight = (min([itemCount, maxRows]) as number) * itemSize;
 
+  const listRef: React.Ref<any> = React.useRef();
```

Comment:
I'd ditch controlled mode and simply expose the method via forwarded ref.
Then, in any component, you could simply do the follownig:

```
const ref = React.useRef<FixedSizeList>();

<FixedSizeList listRef={ref} {...props} />

ref.current.scrollToItem(index);
```
I'm not sure, though, whether react-window executes the `onScroll` handler in such case. If it doesn't, the custom scroll container won't retrieve the dispatched scroll event and hence won't scroll.
You might need to trigger it manually
https://www.npmjs.com/package/react-scrollbars-custom#instance-methods

Reply:
yeah that's fine, that's what I had originally. the prop just seemed cleaner.

> If it doesn't, the custom scroll container won't retrieve the dispatched scroll event and hence won't scroll.

so....... how would I intercept the scroll? wrap it?

Reply:
ooooh hmm hang on that's inverted. it's calling onScroll when it scrolls. hmm

Reply:
> it looks like it's already wrapping the scroll container and adding an onScroll handler :/

Yup.
The question is - is the `onScroll` handler executed or not? I haven't verified that yet, but have you perhaps?

Reply:
the onScroll handler is executed when the _user_ scrolls. I haven't figured out how (if at all) the FixedSizedList emits scrolling events.

I'm like in `ref` hell debugging this. I like have 4 refs now, and I don't know which ones are doing what. 😆

Reply:
the FixedSixedList is trying to set `scrollTop` of `outerRef`

Reply:
> I haven't figured out how (if at all) the FixedSizedList emits scrolling events.

ah, sorry, I didn't mean that. What I was wondering about was whether `onChange` is executed when you call `scrollToItem` method. I assumed it's not and therefore ScrollContainer is not scrolled. You just confirmed that, so we are on the same page now.

> the FixedSixedList is trying to set scrollTop of outerRef

 so... how about wrapping `scrollToItem`?
Our own implementation would do the following:
- call the original impl of react-window's scrollToItem
- measure where we are at (scrollTop might be a good candidate here)
- call scrollToSomething method, but on scroll container this time

Thoughts?